shared/domains/chelonia/chelonia.js

```diff
@@ -798,6 +829,14 @@ export default (sbp('sbp/selectors/register', {
       }
 
       return sbp('chelonia/private/queueEvent', contractID, () => {
+        // This allows us to double-check that the contract is meant to be
+        // removed, as circumstances could have changed from the time remove
+        // was called and this function is executed. For example, `/release`
+        // makes a synchronous check, but processing of other events since
+        // require this to be re-checked (in this case, for reference counts).
+        if (confirmRemovalCallback && !confirmRemovalCallback(contractID)) {
+          return
+        }
```

Comment:
This `confirmRemovalCallback` doesn't seem to be used. Do you have a concrete plan to use it soon? If not then maybe it should be removed until it's needed?

Reply:
What do you mean it's not used? It's used by release

Reply:
Oops, you're right. Sorry for some reason I thought `confirmRemovalCallback` was a named parameter and couldn't find where it was used.